src/main/java/com/mongodb/jdbc/MongoDatabaseMetaData.java

```diff
@@ -1952,165 +1981,111 @@ public boolean autoCommitFailureClosesAllResultSets() throws SQLException {
 
     @Override
     public ResultSet getClientInfoProperties() throws SQLException {
-        ArrayList<MongoResultDoc> rows = new ArrayList<>(4);
+        ArrayList<MongoResultDoc> docs = new ArrayList<>();
```

Comment:
Why did we remove the size hint on this one?

Reply:
I don't think this was intentional. My guess would be that I just deleted the original row and pasted in the new one, which resulted in the size hint getting lost!